examples/common/pigweed/rpc_console/BUILD.gn

```diff
@@ -13,10 +13,26 @@
 # limitations under the License.
 import("//build_overrides/chip.gni")
 import("//build_overrides/pigweed.gni")
+import("$dir_pw_build/python_dist.gni")
 
 group("default") {
   deps = [
-    "py:chip_rpc.install",
+    ":chip_rpc_distribution",
+    ":pip_install_chip_rpc",
+    "py:chip_rpc._build_wheel",
     "py:chip_rpc_wheel",
   ]
 }
+
+pw_create_python_source_tree("chip_rpc_distribution") {
+  packages = [ "py:chip_rpc" ]
+  generate_setup_cfg = {
+    common_config_file = "common_setup.cfg"
+    append_date_to_version = true
```

Comment:
This sounds nondeterministic, can we avoid that please?

Reply:
`append_date_to_version = true` will be needed if this is installed as part of bootstrap. Reason is if there are python code changes pip will think it doesn't need to re-install this package unless the version is bumped. 

The alternative is to install `chip_rpc` as an editable python package. I've got that set now so I'll remove the append_date_to_version option here.